operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -367,11 +367,96 @@ Check the expiration of the certificates.
+         
+         ```bash
+         ncn-m# kubectl delete pod <pod-name> -n <namespace> --force
+         ```
+
+       - Reference to the Shasta v1.3 Admin Guide for any steps related to checking system health.
+
+   After rebooting all the worker nodes, scale nexus replicas back to 1.
+        
+   ```bash
+   ncn-m# kubectl scale deployment nexus -n nexus --replicas=1
+   ```
+
+8. For Shasta v1.3 and earlier, verify vault etcd is healthy.
+
+   1. Check the health of vault etcd.
+      ```bash
+      ncn-m# for pod in $(kubectl get pods -l app=etcd -n vault -o jsonpath='{.items[*].metadata.name}'); do echo "### ${pod} ###"; kubectl -n vault exec $pod  -- /bin/sh -c "ETCDCTL_API=3 etcdctl --cacert /etc/etcdtls/operator/etcd-tls/etcd-client-ca.crt --cert /etc/etcdtls/operator/etcd-tls/etcd-client.crt --key /etc/etcdtls/operator/etcd-tls/etcd-client.key --endpoints https://localhost:2379 endpoint health"; done
+      ```
+
+   2. If the above health of vault etcd reports any pods as `unhealthy`, backup the secret, delete the secret. The operator will create a new secret.
+
+      ```bash
+      ncn-m# kubectl get secret -n vault cray-vault-etcd-tls -o yaml > /root/vault_sec.yaml
+      ncn-m# kubectl delete secret -n vault cray-vault-etcd-tls
+      ```
+ 
+   3. Once the new secret has been created and the cray-vault-etcd pods are running, verify the health of vault etcd.
+
+      ```bash
+      ncn-m#  # kubectl get secret -n vault cray-vault-etcd-tls
+    
+      NAME                  TYPE     DATA   AGE
+      cray-vault-etcd-tls   Opaque   9      5m
+    
+      ncn-m# kubectl get pods -l app=etcd -n vault
+    
+      NAME                         READY   STATUS    RESTARTS   AGE
+      cray-vault-etcd-stzjf6dqd5   1/1     Running   0          10m
+      cray-vault-etcd-ws59fgssxt   1/1     Running   0          10m
+      cray-vault-etcd-xmvfxz48vs   1/1     Running   0          10m
+    
+      ncn-m# for pod in $(kubectl get pods -l app=etcd -n vault -o jsonpath='{.items[*].metadata.name}'); do echo "### ${pod} ###"; kubectl -n vault exec $pod  -- /bin/sh -c "ETCDCTL_API=3 etcdctl --cacert /etc/etcdtls/operator/etcd-tls/etcd-client-ca.crt --cert /etc/etcdtls/operator/etcd-tls/etcd-client.crt --key /etc/etcdtls/operator/etcd-tls/etcd-client.key --endpoints https://localhost:2379 endpoint health"; done
+
+      ### cray-vault-etcd-stzjf6dqd5 ###
+      https://localhost:2379 is healthy: successfully committed proposal: took = 19.999618ms
+      ### cray-vault-etcd-ws59fgssxt ###
+      https://localhost:2379 is healthy: successfully committed proposal: took = 19.597736ms
+      ### cray-vault-etcd-xmvfxz48vs ###
+      https://localhost:2379 is healthy: successfully committed proposal: took = 19.81056ms
+      ```
+
+   4. Vault etcd errors such as `tls: bad certificate." Reconnecting` can be ignored.
```

Comment:
Why is this a step if it's just telling the admin to ignore something?

Reply:
When discussed in the post standup parking lot - it was thought to be worth adding as this relates to old 1.3 issues such as CAST-25011.  So in case this is error is seen and thought to be related to the cert renewal - we can avoid unneeded tickets.